subsys/caf/modules/ble_state.c

```diff
@@ -313,15 +313,11 @@ static void bt_ready(int err)
 	LOG_INF("Bluetooth initialized");
 
 #ifdef CONFIG_CAF_BLE_USE_LLPM
```

Comment:
Please use `if (IS_ENABLED(CONFIG_CAF_BLE_USE_LLPM) {` instead of `#ifdef` if possible after your change is applied. In general we try to avoid `#ifdef` if possible to keep compiler checks (unused could would be opted out anyway)

Reply:
Similarly for `ble_conn_params.c`